static/js/publisher/release.js

```diff
@@ -97,16 +96,14 @@ const initReleases = (id, snapName, releasesData, channelMapsList, options) => {
   const archs = getArchsFromChannelMap(channelMapsList);
 
   ReactDOM.render(
-    <Fragment>
-      <ReleasesController
-        snapName={snapName}
-        releasedChannels={releasedChannels}
-        tracks={tracks}
-        archs={archs}
-        options={options}
-      />
-      <RevisionsList revisions={releasesData.revisions} />
-    </Fragment>,
+    <ReleasesController
```

Comment:
nice

Reply:
Trying to stay sane without adding redux just yet...